### PR TITLE
Updated the repo to have 1 action

### PR DESCRIPTION
**Pull request by yairsimantov20**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "ocean-github-action",
```
Comment (MPTG94) [inline package.json:2]: Name needs to be updated

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "version": "1.0.0",
+  "repository": "git@github.com:port-labs/ocean-github-action.git",
```
Comment (MPTG94) [inline package.json]: Also needs to be updated

File: README.md [hunk 3]
```diff
@@ -1 +1,61 @@
...
+<!-- action-docs-outputs action="action.yml" -->
+
+## Available integration types
+
+All available integration types are listed in the [Ocean integrations library](https://ocean.getport.io/integrations-library/) and can also be found in the data sources menu in the Port UI.
+
+## Example usage
+
+```yaml
+- uses: ocean-github-action/sail@v1
```
Comment (MPTG94) [inline README.md]: Is this the correct call?

**Review by MPTG94 — APPROVED** [review]
Take a look at my comments

File: README.md (hunk 3 above)
Comment (yairsimantov20) [reply README.md]: fair

**Pull request merged**